api/management/commands/ingest_ns_directory.py

```diff
@@ -67,9 +67,9 @@ def postprocessor(path, key, value):
                     }
                     country_directory, _ = CountryDirectory.objects.get_or_create(
                         country=country,
-                        first_name__iexact=data["first_name"],
-                        last_name__iexact=data["last_name"],
-                        position__iexact=data["position"],
```

Comment:
Any reason for this change?

I think caseinsentive search makes sense, Maybe we should update if it is changed instead of creating new?